src/main/java/command/Command.java

```diff
@@ -0,0 +1,34 @@
+package command;
+
+import henry.TaskList;
+
+/**
+ * The command class is the base for all commands.
+ * It only has one method, execute, which returns a CommandResult.
+ * This method is to be implemented by all subclasses of Command.
+ */
+public abstract class Command {
+
+    protected TaskList taskList;
+
+    /**
+     * Returns a CommandResult object based on the type of Command
```

Comment:
I think for void methods which do not return anything, we do not need to add a @return tag for JavaDocs

Reply:
Right, oversight on my part, will be fixed in coming commits.